DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite.utils
 Title: Utility Functions for Open Systems Pharmacology R Packages
-Version: 1.2.0
+Version: 1.2.0.9000
```

Comment:
@IndrajeetPatil I wonder if it's enough or we also need to add
```
development:
  mode: auto
```
https://pkgdown.r-lib.org/reference/build_site.html#development-mode